reviews/models/mixins.py

```diff
@@ -182,17 +195,75 @@ def resubmission_allowed(self, ev):
         return self.reviewable.provider.reviews_workflow == workflow.Workflows.PRE_MODERATION.value
 
     def notify_submit(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        context = self.get_context()
+        context['referrer'] = ev.kwargs.get('user')
+        context['template'] = 'reviews_submission_confirmation'
+        context['is_pre_moderation'] = self.get_info().get('is_pre_moderation')
+        reviews_email.send(context=context)
 
     def notify_accept(self, ev):
-        # TODO email node admins (MOD-53)
-        pass
+        context = self.get_context()
+        context['template'] = 'reviews_submission_status',
+        context['is_pre_moderation'] = self.get_info().get('is_pre_moderation')
+        context['notify_comment'] = self.get_info().get('notify_comment')
+        context['is_rejected'] = self.get_info().get('is_rejected')
```

Comment:
Better to call `get_info` once and put the result in a variable, rather than calling it over and over.